modules/network-observability-filtering-ebpf-rule.adoc

```diff
@@ -27,25 +30,32 @@ spec:
   namespace: netobserv
   deploymentModel: Direct
   agent:
-    type: eBPF
-    ebpf:
-      flowFilter:
-        action: Accept  <1>
-        cidr: 172.210.150.1/24 <2>
-        protocol: SCTP
-        direction: Ingress
-        destPortRange: 80-100
-        peerIP: 10.10.10.10
-        enable: true    <3>
+  type: eBPF
+  ebpf:
+    flowFilter:
+      enable: true <1>
+      rules:
+        - action: Accept <2>
+          cidr: 172.30.0.0/16 <3>
+          peerCIDR: 10.128.0.0/14
+          sampling: 10
+        - action: Accept
+          cidr: 10.128.0.0/14
+          peerCIDR: 10.128.0.0/14
+          sampling: 20
+        - action: Accept
+          cidr: 8.8.8.8/32
+          sampling: 30
+          peerCIDR: 10.128.0.0/14
```

Comment:
since these examples provide overlapping CIDRs, I am not sure if it's allowed to have overlapping CIDRs and what would be the behavior in that case cc @Amoghrd

Reply:
Duplicate peerCIDR's are allowed. Duplicate CIDR's are not allowed. So should be good